packages/cli/create-strapi-app/src/utils/database.ts

```diff
@@ -107,7 +107,7 @@ export async function getDatabaseInfos(options: Options): Promise<DBConfig> {
 const sqlClientModule = {
   mysql: { mysql2: '3.9.4' },
   postgres: { pg: '8.8.0' },
-  sqlite: { 'better-sqlite3': '9.4.3' },
+  sqlite: { 'better-sqlite3': '11.3.0' },
```

Comment:
No breaking changes to report ?

Reply:
This is just the version we generate new projects with, it wouldn't break any existing code, and old versions still work with Node 20. But if we generate new projects with a later version, the user doesn't have to figure out why node 22 doesn't work.

Reply:
as long as v11 is compat with node 18 & 20 it's all good 👍

Reply:
Yep, the only breaking changes are dropping support for versions of Node we don't support (node 21 and node <= 14 I think)